exec/java-exec/src/main/java/org/apache/drill/exec/expr/ClassGenerator.java

```diff
@@ -66,7 +68,17 @@
   public static final GeneratorMapping DEFAULT_CONSTANT_MAP = GM("doSetup", "doSetup", null, null);
 
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(ClassGenerator.class);
-  public static enum BlockType {SETUP, EVAL, RESET, CLEANUP};
+
+  /**
+   * Field has 2 indexes within the constant pull: field item + name and type item.
```

Comment:
I missed the third index. Fixed estimation of max index value, thanks. Also added comment that explains the way, how it is calculating.